src/loop_util/loop_util.c

```diff
@@ -0,0 +1,137 @@
+/* Copyright (c) 2015 CoreOS. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+#include <sys/types.h>
+#include <sys/mount.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+#include <linux/loop.h>
+#include <sys/file.h>
+#include <libmount/libmount.h>
+
+/*#include "path-util.h"
+#include "loopback-setup.h"
+#include "dev-setup.h"
+#include "fdset.h"*/
+
+
+static int get_node(const char *image_path, char *source) {
```

Comment:
source is unused, it could be used to return loopdev as a string instead of an integer. either scheme works.